multichain-testing/tools/e2e-tools.js

```diff
@@ -106,20 +105,34 @@ export const makeBlockTool = ({ rpc, delay }) => {
  * @param {string} [opts.bundleId]
  */
 const installBundle = async (fullPath, opts) => {
-  const { id, agd, progress = console.log } = opts;
+  const { id, agd, delay, progress = console.log } = opts;
   const { chainId = 'agoriclocal', installer = 'faucet' } = opts;
-  const from = await agd.lookup(installer);
+  const from = agd.lookup(installer);
   // const explainDelay = (ms, info) => {
   //   progress('follow', { ...info, delay: ms / 1000 }, '...');
   //   return delay(ms);
   // };
   // const updates = follow('bundles', { delay: explainDelay });
   // await updates.next();
-  const tx = await agd.tx(['swingset', 'install-bundle', `@${fullPath}`], {
-    from,
-    chainId,
-    yes: true,
-  });
+  const installBundle = () =>
+    agd.tx(['swingset', 'install-bundle', `@${fullPath}`], {
+      from,
+      chainId,
+      yes: true,
+    });
+  let tx;
+  try {
+    tx = await installBundle();
+  } catch (err) {
+    console.warn('WARN: retrying failed install-bundle', err);
+    // TODO make available from client-utils
+    const { waitForBlock } = makeBlockTool({
+      rpc: makeHttpClient(rpcUrl, global.fetch),
```

Comment:
```suggestion
      rpc: makeHttpClient(rpcUrl, globalThis.fetch),
```

> Stability: 3 - Legacy. Use globalThis instead.
 -- https://nodejs.org/api/globals.html#global

Reply:
Given that several other IO capabilities are injected as `opts`, it seems inconsistent to use a global `fetch`.

But... anything that works is better than this flake.